src/runtime/runtime.cs

```diff
@@ -193,8 +177,45 @@ internal static void Initialize(bool initSigs = false)
             PyScopeManager.Reset();
             ClassManager.Reset();
             ClassDerivedObject.Reset();
-            TypeManager.Reset();
+            TypeManager.Initialize();
+
+            InitPyMembers();
+
+            // Initialize data about the platform we're running on. We need
+            // this for the type manager and potentially other details. Must
+            // happen after caching the python types, above.
+            NativeCodePageHelper.InitializePlatformData();
 
+            // Initialize modules that depend on the runtime class.
+            AssemblyManager.Initialize();
+#if !NETSTANDARD
```

Comment:
Why is this not applicable to .NET Standard build?

Reply:
Domain operation is not fully supported by .NET Standard now.

Reply:
We should avoid compile-time checks, as we eventually want to produce a single .NET Standard DLL. Can you replace this with a runtime check for `AppDomain` unloading support.